services/inference/tsfminference/inference.py

```diff
@@ -43,7 +43,7 @@ def forecast(self, input: ForecastingInferenceInput):
 
         if ex is not None:
             detail = error_message(ex)
-            LOGGER.error(detail)
+            LOGGER.exception(detail)
```

Comment:
I think you want `LOGGER.exception(ex)` instead?

Reply:
Yep, fixed it.